dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -1042,7 +1042,14 @@ def auto_score!
         free_lunch_percent = responses[:principal_free_lunch_percent].present? ?
                                responses[:principal_free_lunch_percent].to_i :
                                school_stats&.frl_eligible_percent
-        free_lunch_percent_cutoff = school_stats&.rural_school? ? 40 : 50
+        free_lunch_percent_cutoff =
+          if regional_partner&.frl_guardrail_percent
+            regional_partner&.frl_guardrail_percent.to_i
+          elsif school_stats&.rural_school?
+            40
```

Comment:
as a cleanup, could you move these into variables, such as `DEFAULT_RURAL_FRL_GUARDRAIL`? it helps for readability in the future but doesn't need to block this PR as there's some time pressure here

Reply:
Will do! I'll make this a change to go into staging-next